environment/config.go

```diff
@@ -51,6 +55,34 @@ type Configuration struct {
 
 	// StaticConfiguration is used for running M3DB with a static config
 	Static *StaticConfiguration `yaml:"static"`
+
+	// Presence of a (etcd) server in this config denotes an embedded cluster
+	EmbeddedServer *EmbeddedKV `yaml:"embeddedServer"`
+
+	// NamespaceTimeout is the timeout duration for setting a namespace
```

Comment:
For setting a namespace?  Do you mean namespace creation or resolving a namespace from KV?